package.json

```diff
@@ -11,11 +11,12 @@
     "build": "vue-cli-service electron:build",
     "postinstall": "electron-builder install-app-deps",
     "postuninstall": "electron-builder install-app-deps",
-    "lint": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter src",
-    "lint:fix": "eslint --ext .js,.vue -f ./node_modules/eslint-friendly-formatter --fix src",
+    "prepare": "ncp src/originalBudgetTrackingApp/categoryCalculationScript-example.js src/originalBudgetTrackingApp/categoryCalculationScript.js",
```

Comment:
I think ncp is an overkill here, cp -n will do the trick

Reply:
`cp -n` shouldn't work on Windows, and also the `|| true` syntax, I don't know how it works on *Github Actions*.

On my desktop, it didn't work.

Reply:
ah, forgot about windows... Ok, will change that